test-cases/tests/documentation/i01-verify-release-notes.md

```diff
@@ -1,10 +1,10 @@
 ---
```

Comment:
I would rather avoid deleting RHMI test cases, if we could use them in a future.

So if you think this test case might be still valid for RHMI in a future, you can split this test case to 2 similar test case files (1 for RHMI, 1 for RHOAM):

```
# For RHMI
test-cases/tests/documentation/i01a-verify-release-notes.md
# For RHOAM
test-cases/tests/documentation/i01b-verify-release-notes.md
```

Also note that the **title** needs to be updated in those test cases
For RHMI:
**# I01A - Verify release notes**
for RHOAM:
**# I01B - Verify release notes**

Reply:
I don't think we are creating release notes for RHMI anymore.